Resources/config/services.xml

```diff
@@ -4,84 +4,52 @@
+        <service class="ETSGlobal\LogBundle\Monolog\Processor\ChangeExceptionsLogLevelProcessor" public="false">
             <argument type="string">%ets_global_log.custom_exceptions_levels%</argument>
             <argument type="string">%ets_global_log.http_exceptions_levels%</argument>
 
             <tag name="monolog.processor" />
         </service>
 
-        <service id="ets_global_log.monolog.processor.token_collection" class="%ets_global_log.monolog.processor.token_collection.class%" public="false">
-            <argument type="service" id="ets_global_log.tracing.token_collection" />
-
+        <service class="ETSGlobal\LogBundle\Monolog\Processor\TokenCollectionProcessor" public="false">
             <tag name="monolog.processor" />
         </service>
 
-        <service id="ets_global_log.monolog.processor.extra_field" class="%ets_global_log.monolog.processor.extra_field.class%" public="false">
+        <service class="ETSGlobal\LogBundle\Monolog\Processor\ExtraFieldProcessor" public="false">
             <argument type="string">application</argument>
             <argument type="string">%ets_global_log.app_name%</argument>
 
             <tag name="monolog.processor" />
         </service>
 
-        <service id="ets_global_log.monolog.formatter.token_collection" class="%ets_global_log.monolog.formatter.token_collection.class%" public="false">
-            <argument type="service" id="ets_global_log.tracing.token_collection" />
+        <service class="ETSGlobal\LogBundle\Monolog\Formatter\TokenCollectionFormatter" public="false">
             <argument type="string">%ets_global_log.log_format%</argument>
         </service>
 
-        <service id="ets_global_log.monolog.handler.content_data_modifier.add_class_line_context" class="%ets_global_log.monolog.handler.content_data_modifier.add_class_line_context.class%" public="false"/>
+        <service class="ETSGlobal\LogBundle\Monolog\Handler\ContentDataModifier\AddClassLineContext" public="false" />
 
-        <service id="ets_global_log.monolog.handler.content_data_modifier.add_jira_link" class="%ets_global_log.monolog.handler.content_data_modifier.add_jira_link.class%" public="false">
+        <service class="ETSGlobal\LogBundle\Monolog\Handler\ContentDataModifier\AddJiraLink" public="false">
             <argument type="string">%ets_global_log.handlers.slack.jira_url%</argument>
         </service>
 
-        <service id="ets_global_log.monolog.handler.content_data_modifier.add_kibana_token_filter_links" class="%ets_global_log.monolog.handler.content_data_modifier.add_kibana_token_filter_links.class%" public="false">
+        <service class="ETSGlobal\LogBundle\Monolog\Handler\ContentDataModifier\AddKibanaTokenFilterLinks" public="false">
             <argument type="string">%ets_global_log.handlers.slack.kibana_url%</argument>
-            <argument type="service" id="ets_global_log.tracing.token_collection" />
```

Comment:
why?

Reply:
Autowiring (I forgot to add it in beginig of declaration)